AzureCalling/app/src/main/java/com/azure/samples/communication/calling/external/calling/CallingContext.java

```diff
@@ -401,7 +410,11 @@ private void callWithOptions(
         final JoinCallOptions joinCallOptions = new JoinCallOptions();
         joinCallOptions.setVideoOptions(videoOptions);
         joinCallOptions.setAudioOptions(audioOptions);
-        call = agent.join(appContext, groupCallLocator, joinCallOptions);
+        try {
+            call = agent.join(appContext, groupCallLocator, joinCallOptions);
+        } catch (Exception e) {
+            throw e;
+        }
```

Comment:
If `agent.join` is throwing a RuntimeException, we should not need all the try/catch blocks.

Reply:
Got it!